cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
-            <li><i>%(MODE_BRIGHT)s</i>: Fit the spline to the lightest pixels
-            in the image, excluding the darker pixels. This may be appropriate
-            for a histologically stained image.</li>
-            <li><i>%(MODE_GRAY)s</i>: Fit the spline to mid-range pixels,
-            excluding both dark and light pixels. This may be appropriate
-            for a brightfield image where the objects of interest have
-            light and dark features.</li></ul>""" % globals())
+            "Background mode",
+            [MODE_AUTO, MODE_DARK, MODE_BRIGHT, MODE_GRAY], doc="""\              
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method
+and spline parameters are not calculated automatically)*
+This setting determines which pixels are background and which are
+foreground.
+
+-  *%(MODE_AUTO)s*: Determine the mode from the image. This will set
+   the mode to %(MODE_DARK)s if most of the pixels are dark,
+   %(MODE_BRIGHT)s if most of the pixels are bright and %(MODE_GRAY)s
+   if there are relatively few dark and light pixels relative to the
+   number of mid-level pixels
+-  *%(MODE_DARK)s*: Fit the spline to the darkest pixels in the image,
+   excluding brighter pixels from consideration. This may be appropriate
+   for a fluorescent image.
+-  *%(MODE_BRIGHT)s*: Fit the spline to the lightest pixels in the
+   image, excluding the darker pixels. This may be appropriate for a
+   histologically stained image.
+-  *%(MODE_GRAY)s*: Fit the spline to mid-range pixels, excluding both
+   dark and light pixels. This may be appropriate for a brightfield
+   image where the objects of interest have light and dark features.
+""" % globals())
 
         self.spline_threshold = cps.Float(
-                "Background threshold", 2, minval=.1, maxval=5.0, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines the cutoff used when excluding
-            foreground pixels from consideration. On each iteration,
-            the method computes the standard deviation of background
-            pixels from the computed background. The number entered in this
-            setting is the number of standard deviations a pixel can be
-            from the computed background on the last pass if it is to
-            be considered as background during the next pass.
-            <p>
-            You should enter a higher number to converge stabily and slowly
-            on a final background and a lower number to converge more
-            rapidly, but with lower stability. The default for this
-            parameter is two standard deviations; this will provide a fairly
-            stable background estimate.""" % globals())
+            "Background threshold", 2, minval=.1, maxval=5.0, doc="""\
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.